### DISCUSSION TITLE
413 Request Entity Too Large

### DISCUSSION THREAD

**Discussion by laxmiigupta** [wardenenv/warden#636] [Help & Support]
getting this error then we save product & banner 
![image](https://user-images.githubusercontent.com/127080319/228858261-0de527fc-9899-4cee-a149-a657ef482894.png)

**Comment by atty31** [comment]
I m getting the same, did you find a fix for this ?

**Comment by sprankhub** [comment]
See https://github.com/orgs/wardenenv/discussions/632#discussioncomment-15075483.
